### DISCUSSION TITLE
Timeframe for next release?

### DISCUSSION THREAD

**Discussion by RetroCoder** [alirezanet/Gridify#228] [General]
My company is looking to move from Sieve to Gridify but needs some of the functionality in the 2.15.0 preview release. Unfortunately, they won’t ship with a preview version, so I am stuck using Sieve until you produce the production ready 2.15.0 release. Do you have an expected release date? Obviously, it is difficult to be specific so would that be this year or early next year?

**Comment by alirezanet** [comment]
Hi @RetroCoder,
Yeah we're planning to release it soon, (this or next month probably)

**Comment by alirezanet** [comment]
https://github.com/alirezanet/Gridify/releases/tag/v2.15.0
